.golangci.yml

```diff
@@ -4,6 +4,8 @@ run:
     - extern
     - integration/repdao
     - integration/repdao_dp
+  skip-files:
+    - pkg/net/host.go # this is failing for me because libp2p cannot be found on go1.20.1
```

Comment:
very weird - will this cause any issues with deployment though?

Reply:
it shouldnt. Both version 1.20.1 and 1.21.X were causing me linter errors locally because of some thing in libp2p that isnt supported in the new versions. The container we ship has the right version of go.